packages/governance/test/unitTests/test-committee.js

```diff
@@ -74,7 +74,7 @@ test('committee-open question:one', async t => {
     electionType: ElectionType.SURVEY,
     maxChoices: 1,
     closingRule: {
-      timer: buildManualTimer(console.log),
+      timer: buildManualTimer(t.log),
```

Comment:
Console pollution for successful tests really irks me; it makes actual failures scroll off the screen.

Reply:
I think the solution in both of our cases is to buy a bigger scrollback buffer. :)

Reply:
I agree with your change to gov-collateral, but I find that the manualTimers are not noisy enough to warrant sending them anywhere but `t.log`.

Reply:
ah, yeah me too. didn't realize that `t.log` only prints if the test fails.

though because not all logging has access to `t.log` this change makes it so the stream is messages is separated into two different places, making it harder to follow the messages.

what I've done in the past to keep logs clean is override `console` methods with a buffer. we could do something for Ava like that which only prints the console if the test fails. WDYT?

for now humbling requesting that this change be separated to another PR where it can be considered more holistically, but won't block on it.